source/fetch_pbd.rst

```diff
@@ -0,0 +1,131 @@
+This behaviour is a bit complicated. It is recommended that you specify the full path to files or else it will look in your .ros folder.
+If you specify a from_file then actions will be loaded from that file. They will replace the ones in your session database.
+Whatever was in your session database will get stored in a timestamped file in your .ros folder (not overwritten).
+If you specify a to_file then whatever is in your current session file be saved to that file.
+
+::
+
+    >$ source ~/catkin_ws/devel/setup.bash
+    >$ roslaunch fetch_pbd_interaction pbd.launch from_file:=/full/path/from.json to_file:=/full/path/to.json
+
+Using the GUI
+-------------
+
+In your browser go to ROBOT_HOSTNAME:8080 in your browser to use the GUI. This can be used on mobile as well. The mobile version does not show the visualizer, but can be useful for saving/deleting
+
+The main page lists all the available actions.
+
+.. figure:: _static/fetch_pbd_home_screen.png
+   :width: 100%
+   :align: center
+   :figclass: align-centered
+
+You can directly run/copy/delete actions from the main page. Or hit the "Edit" button to see more information on that action.
+
+On the "Current Action" screen, most of the buttons are pretty self-explanatory. You can execute the entire action using the "Run" button at the bottom of the screen. This will execute all of the primitives in the order they appear in the Primitive List. You can click on a specific primitive (either the marker or the list item), to highlight the primitive.
+
+.. figure:: _static/fetch_pbd_select.png
+   :width: 100%
+   :align: center
+   :figclass: align-centered
+
+You can show/hide the markers for each primitive by clicking the marker icon for the primitive in the Primitive List.
+
+.. figure:: _static/fetch_pbd_hide_marker.png
+   :width: 100%
+   :align: center
+   :figclass: align-centered
+
+You can change the order of the primitives by dragging them to a new position in the list.
+
+.. figure:: _static/fetch_pbd_reorder.png
+   :width: 100%
+   :align: center
+   :figclass: align-centered
+
+You can edit the position and orientation of certain primitives by clicking the edit icon or by moving the interactive marker.
+
+.. figure:: _static/fetch_pbd_edit.png
+   :width: 100%
+   :align: center
+   :figclass: align-centered
+
+You can change the frame that certain primitives are relative to by right-clicking the marker.
+
+.. figure:: _static/fetch_pbd_change_frame.png
+   :width: 100%
+   :align: center
+   :figclass: align-centered
+
+You can also change the name of the action.
+
+.. figure:: _static/fetch_pbd_rename.png
+   :width: 100%
+   :align: center
+   :figclass: align-centered
+
+Code Interface
+--------------
+
+You can also access the actions you've programmed through code. You still need to run pbd_backend.launch.
+
+::
+
+    >$ source ~/catkin_ws/devel/setup.bash
+    >$ rosrun fetch_pbd_interaction demo.py
+
+System Overview
+---------------
+
+Interaction Node:
```

Comment:
To make these block headings stand out like they do in Markdown, either make them bold, or do
```restructuredtext
Interaction Node
===============

Lots of text
```

(the key thing is that the underlining character shoudl be different from the main section headings)